changelog/10908.bugfix.md

```diff
@@ -0,0 +1 @@
+Pin requirement for `scipy~=1.8.0` since `scipy==1.8.0` is not backward compatible with `scipy~=1.7.0`.
```

Comment:
```suggestion
Pin requirement for `scipy<1.8.0` since `scipy==1.8.0` is not backward compatible with `scipy<1.8.0` and additionally requires Python>=3.8, while Rasa supports Python 3.7 as well.
```